pkg/go.mod

```diff
@@ -57,7 +57,7 @@ require (
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	google.golang.org/api v0.74.0
 	google.golang.org/genproto v0.0.0-20220405205423-9d709892a2bf
-	google.golang.org/grpc v1.49.0
```

Comment:
There are a few downgrades here, is this intentional?

Reply:
Good catch! I'll double check.

Reply:
This was a mistaken change, reverted.